src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -859,7 +859,10 @@ const StorageRow = ({ connectionName, allowNoDisk, storageSize, storageSizeUnit,
                 <FormSelect id="storage-pool-select"
                             value={storagePoolName}
                             onChange={e => onValueChanged('storagePool', e)}>
-                    <FormSelectOption value="NewVolume" key="NewVolume" label={_("Create new volume")} />
+                    <FormSelectOptionGroup key="Create new volume" label={_("Create new volume")}>
+                        <FormSelectOption value="NewVolumeQCOW2" key="NewVolumeQCOW2" label={_("Qcow2 volume")} />
```

Comment:
It's weird to title case Qcow2.
Maybe use something like: 

format: qcow2
format: raw

@garrett ?

Reply:
Answered in my review. (And I agree the case is wrong.)